lib/setup/setupPacketManager.js

```diff
@@ -21,6 +21,7 @@ class PacketManager {
         this.manager = (options && options.manager) || '';
         this.logger = (options && options.logger) || {
             silly: text => options.logLevel >= LOG_LEVELS.silly && console.log(text),
+            info:  text => options.logLevel >= LOG_LEVELS.info   && console.log(text),
```

Comment:
I think we then should also add debiug so that it is as usual ... what do you think?

Reply:
It's there, it's just not ordered. https://github.com/ioBroker/ioBroker.js-controller/pull/656/files#diff-fe59b99ac67ab90f2c4f1fb6b4ef5797L27

Reply:
info was missing, then I though about replacing 'log' by 'info' but now everything is there which has been defined by bf in https://github.com/ioBroker/ioBroker.js-controller/pull/656/files#diff-fe59b99ac67ab90f2c4f1fb6b4ef5797L7-L14

Reply:
For clarification, 'info' had to be added, because this.logger.info had been used in the code and thus produced an error `TypeError: this.logger.info is not a function`

Reply:
I exactly feared that when I first commented to have all methods there :-) SO great, thank you!